gudrun_classes/container.py

```diff
@@ -83,8 +89,19 @@ def __str__(self):
             ''
             )
 
-        return (
+        if self.densityUnits == UnitsOfDensity.ATOMIC.value:
+            units = 'atoms/\u212b^3'
+            density = self.density*-1
```

Comment:
Such a small suggestion, but why not:
```suggestion
            density = -self.density
```